Extensions/blacklist.js

```diff
@@ -1174,6 +1178,287 @@ XKit.extensions.blacklist = new Object({
 		$("#xkit-extensions-panel-right").nanoScroller();
 		$("#xkit-extensions-panel-right").nanoScroller({ scroll: 'top' });
 
+		XKit.extensions.blacklist.nativeExportCpanel(m_div);
+	},
+
+	nativeExportCpanel: function(m_div) {
+
+		/**
+		 * Create elements with simple syntax
+		 *
+		 * @param {string} tagName - Type of element to create
+		 * @param {object} [attributes] - Property-value pairs to set as HTML/XML attributes (e.g. { href: '/' })
+		 * @param {object} [events] - Property-value pairs to set as event listeners (e.g. { click: () => {} })
+		 * @param {(Node|string)[]} [children] - Zero or more valid children
+		 * @returns {Element} Element created to specification
+		 */
+		const dom = (tagName, attributes = {}, events = {}, children = []) => {
```

Comment:
I added some more jquery-like suggestions inline but if we want to keep this method we should at least remove the complex SVG stuff that we're not using (especially since we're not using setAttributeNs so the SVG stuff only half-works), and maybe name it something slightly more descriptive like `createElement`.

Reply:
Yeah, I just copied the XKit Rewritten generic one literally. The way I see it is that, big picture, if we were committing to overhaul this codebase, I would add some variant of this to xkit patches, in which case the full version is appropriate. If we aren't (and I am not), then we're doing spot edits.

I can migrate to jquery, just have to remember how it works.

Reply:
I don't think "overhauling the entire codebase" or "just doing spot edits" are the only two options. jQuery works fine for this codebase, and it's better to be consistent in using jQuery for most things then it is to have some parts of the codebase using something different then other parts of the codebase based on when they were written. It's better to move forward gradually without changing too many big things unnecessarily. I also just don't really like the API design of this helper (it requires `null` parameters for almost every common use-case`).